bin/src/Flags.purs

```diff
@@ -69,9 +69,16 @@ outfile =
   OT.optional
     $ O.strOption
         ( O.long "outfile"
-            <> O.help "Destination path for the bundle"
+            <> O.help "Destination path for the bundle. Defaults to ./index.js"
         )
 
+forceOverwrite :: Parser Boolean
+forceOverwrite =
+  O.switch
+    ( O.long "force-overwrite"
+        <> O.help "Overwrite the target bundle file even if it was manually created or generated by a tool other than Spago."
+    )
+
```

Comment:
I think we could shorten this to `--force`, while at the same time having a more specific name to imply the flag is only used for bundling (as other parts of the CLI could use `--force` for other things)
```suggestion
forceBundle :: Parser Boolean
forceBundle =
  O.switch
    ( O.long "force"
        <> O.help "Overwrite the target bundle file even if it was manually created or generated by a tool other than Spago."
    )

```

Reply:
Done